src/audio/module_adapter/module/waves.c

```diff
@@ -342,7 +342,7 @@ static int waves_effect_init(struct processing_module *mod)
 	waves_codec->o_format = waves_codec->i_format;
 
 	waves_codec->sample_size_in_bytes = sample_bytes;
-	waves_codec->buffer_samples = (src_fmt->rate * 2) / 1000; /* 2 ms io buffers */
+	waves_codec->buffer_samples = (src_fmt->rate * 1) / 1000; /* 1 ms io buffers */
```

Comment:
Is there an option to use pipeline processing period parameter from topology instead of having static 1ms buffer definition?

Reply:
ack, this should really come from topology (and align with the other parts of the topology).